programs/pluto/linux-copy/linux/xfrm.h

```diff
@@ -519,6 +519,12 @@ struct xfrm_user_offload {
  */
 #define XFRM_OFFLOAD_IPV6	1
 #define XFRM_OFFLOAD_INBOUND	2
```

Comment:
add a comment this requires linux kernel 6.2+

Reply:
Ack